sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/arm/collection/implementation/CreatableResourcesImpl.java

```diff
@@ -66,6 +66,7 @@ public final Flux<T> createAsync(List<? extends Creatable<T>> creatables) {
             .flatMapMany(rootResource -> Flux.fromIterable(rootResource.createdTopLevelResources()));
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Agent java version has upgraded from `17.0.5` to `17.0.6` and seems `-Werror` somehow got enabled by default:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2157970&view=logs&j=e70f4596-c30d-5c1e-6b94-a902b89515b5&t=e6dd3e0b-9ba5-5875-a9f3-7b940fa1068a&l=31